### PR TITLE
Autocomplete slot names.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-editor-service/100)

### CODE REVIEW

File: src/language-server/auto-completer.ts [hunk 1]
```diff
@@ -241,6 +245,31 @@ export default class AutoCompleter extends Handler {
...
+      kind: 'element',
+      id: parent.tagName,
+      imported: true,
+      externalPackages: true
+    });
+    if (!parentDefn) {
+      return {isIncomplete: false, items: []};
+    }
+    const items =
+        parentDefn.slots.map(sl => sl.name).filter(name => !!name).map(name => {
```
Comment (TimvdLippe) [inline src/language-server/auto-completer.ts]: Maybe also filter the already declared slots? E.g. all sibling slot names?

Comment (rictic) [reply src/language-server/auto-completer.ts]: Oh! Good idea

Comment (rictic) [reply src/language-server/auto-completer.ts]: Hm, wait, I'm pretty sure you can distribute multiple nodes to the same slot. This came up with the paper-toolbar migration.

Comment (rictic) [reply src/language-server/auto-completer.ts]: Yeah, distributing multiple nodes to the same slot is allowed: https://jsbin.com/tekodehixo/1/edit?html,js,output

**Pull request merged**